pkg/api/openshiftcluster_test.go

```diff
@@ -49,3 +52,41 @@ func TestIsTerminal(t *testing.T) {
 		})
 	}
 }
+
+func TestIsWimi(t *testing.T) {
+	tests := []*struct {
```

Comment:
Can you please also add 2 more test cases here, although this is not going to happen, but to cover all the cases for non-miwi?:-
1. Both PlatformWorkloadIdentityProfile and ServicePrincipalProfile are nil. want - false
2. Both PlatformWorkloadIdentityProfile and ServicePrincipalProfile are set. want - false

Reply:
Done, thank you for the suggestion.